doc/source/prometheus.rst

```diff
@@ -0,0 +1,31 @@
+====================
+ Prometheus support
+====================
+
+`Prometheus`_ can use Gnocchi to store its data through
+`Remote Write Adapter`_.
+
+Example of Prometheus configuration::
+
+  remote_write:
+  - url: "http://localhost:8041/v1/prometheus/write"
+
+
+The `/v1/prometheus/write` handles the `WriteRequest` protobuf message
```

Comment:
The `/v1/prometheus/write` endpoint handles the `WriteRequest` protobuf message

Reply:
done